### PR TITLE
add 'bit(a,b)' math function

### PR DESCRIPTION
**Pull request by havardpe**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

bit(a,b) is similar to (a & (1 << b))
'a' should be an integer in the range [-128,127]
'b' should be an integer in the range [0,7]
conformance testing of bit(a,b) is disabled until we have an implementation in Java as well.

@arnej27959 please review
@lesters @bratseth FYI

### CODE REVIEW

File: eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp [hunk 1]
```diff
@@ -29,6 +29,12 @@ double vespalib_eval_approx(double a, double b) { return (vespalib::approx_equal
 double vespalib_eval_relu(double a) { return std::max(a, 0.0); }
 double vespalib_eval_sigmoid(double a) { return 1.0 / (1.0 + std::exp(-1.0 * a)); }
 double vespalib_eval_elu(double a) { return (a < 0) ? std::exp(a) - 1.0 : a; }
+double vespalib_eval_bit(double a, double b) {
+    // must match Bit::f
```
Comment (arnej27959) [inline eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp:33]: it would be easier to just call Bit::f here

**Review by arnej27959 — APPROVED** [review]
looks good to me

File: eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp (hunk 1 above)
Comment (havardpe) [reply eval/src/vespa/eval/eval/llvm/llvm_wrapper.cpp:33]: I considered it, but did not think gcc would be able to inline it (feels bad to get 2 function calls per bit). Maybe we should have a common inline implementation that we call from both places.

**Comment by havardpe** [comment]
@arnej27959 please take another look

**Comment by havardpe** [comment]
` 10:54:33 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-install-plugin:2.5.2:install (default-install) on project bundle-plugin: Failed to install metadata com.yahoo.vespa/maven-metadata.xml: Error updating group repository metadata: in epilog non whitespace content is not allowed but got / (position: END_TAG seen ...</metadata>\n/... @16:2) -> [Help 1] `

**Review by arnej27959 — APPROVED** [review]
👍

**Pull request merged**